### DISCUSSION TITLE
【文本编辑器/deepin-editor】支持 F3 通用快捷键以在查找结果中进行上一个/下一个切换

### DISCUSSION THREAD

**Discussion by itsXuSt** [linuxdeepin/developer-center#3334] [Features Request & Ideas | 特性请求 & 头脑风暴]
如标题所言，F3/Shift F3 为绝大多数编辑器都支持的用于切换搜索结果中的上一项/下一项，但深度编辑器不支持，只能用回车，用回车有个弊端就是焦点必须在搜索输入框中才能进行，否则变成了文本编辑器的输入。

as title said, F3/Shift F3 is supported by most of editors to switch the next and previous item, but it's not supported by deepin-editor.
Enter can be used but only when the focus is in the lineedit it works.

**Comment by BLumia** [comment]
@meiyixiang
